agents/aggregator/aggregator_agent.py

```diff
@@ -32,13 +33,16 @@ class Provider:
             Start time of current frame
         blocks (dict):
             All blocks that are written by provider.
+        aliases (dict):
+            Alias map from stream name to list of aliases
```

Comment:
Is "stream name" the official name for a uniquely named vector of data in a feed?  We have been using "field" in sisock.  I would prefer to use field or channel, generally.

Reply:
I'm ok switching it to `field`.